### PR TITLE
fix: update the doc for metrics cardinality enforcement

### PR DESCRIPTION
**Pull request by yongruilin**
### Description
This PR correct the doc for metrics cardinality enforcement
1. Use correct flag name as `allow-metric-labels` following the [flag definition](https://github.com/kubernetes/kubernetes/blob/c15581b277e9e42d830898357fdbd92dcf7981c6/staging/src/k8s.io/component-base/metrics/options.go#L81)
2. Use correct manifest pattern following the [flag process function](https://github.com/kubernetes/kubernetes/blob/c15581b277e9e42d830898357fdbd92dcf7981c6/staging/src/k8s.io/component-base/metrics/opts.go#L365 )

### Issue

Closes: #

### CODE REVIEW

File: content/en/docs/concepts/cluster-administration/system-metrics.md [hunk 1]
```diff
@@ -193,23 +193,22 @@ Each mapping is of the format `<metric_name>,<label_name>=<allowed_labels>` wher
 The overall format looks like:
 
 ```
---allow-label-value <metric_name>,<label_name>='<allow_value1>, <allow_value2>...', <metric_name2>,<label_name>='<allow_value1>, <allow_value2>...', ...
+--allow-metric-labels <metric_name>,<label_name>='<allow_value1>, <allow_value2>...', <metric_name2>,<label_name>='<allow_value1>, <allow_value2>...', ...
```
Comment (tengqm) [inline content/en/docs/concepts/cluster-administration/system-metrics.md:196]: Line 186 should be changed as well.

**Review by dipesh-rawat — COMMENTED** [review]
/lgtm

**Pull request merged**